tests/phpunit/includes/tests-traversal.php

```diff
@@ -668,6 +668,9 @@ private function _test_find_base( $variant_id, $options, $query_fields = false )
 
 		$this->assertEquals( (string) $data['id'], (string) $p->id(), sprintf( 'Item ID not as expected (%s) [%s]', $data['field_id'], $variant_id ) );
 
+		$this->assertEquals( 1, (int) $p->field( '_total' ), sprintf( 'Total value not as expected (%s) %s [%s]', $p->total(), var_export( $p->field( '_total' ), true ), $variant_id ) );
```

Comment:
It's a really small thing and this is in the tests, but I think we've switched to consistently using numbered placeholders for `sprintf` in all cases?  @GaryJones ?

Reply:
Actually looking up where we did it, this might only be for translatable strings

Reply:
The general rule is - if it's more than a single placeholder _in a translatable string_, then yes, it should be numbered, since translators for other locales may need to re-arrange them.

In this case, the string isn't translatable, and the placeholder fillers are not re-used, so there's no technical benefit to using numbered placeholders here.

As an aside, my _personal_ preference (which would seem to go against the current code style in this project), is to lay out sprintf args on new lines, since it makes it easier to read (and also easier to see if sufficient escaping has been added). So instead of:

```php
$this->assertEquals( 1, (int) $p->field( '_total' ), sprintf( 'Total value not as expected (%s) %s [%s]', $p->total(), var_export( $p->field( '_total' ), true ), $variant_id ) );
```

I'd say it was more readable to use:

```php
$message = sprintf(
	'Total value not as expected (%s) %s [%s]',
	$p->total(),
	var_export( $p->field( '_total' ), true ),
	$variant_id
);

$this->assertEquals( 1, (int) $p->field( '_total' ), $message );
```

which doesn't just have a long line of commas and parentheses.